src/Storages/StorageURL.cpp

```diff
@@ -457,7 +457,7 @@ std::pair<Poco::URI, std::unique_ptr<ReadWriteBufferFromHTTP>> StorageURLSource:
 
         const auto settings = context_->getSettings();
 
-        auto proxy_config = getProxyConfiguration(http_method);
```

Comment:
This one is on me. In some places, the protocol is called scheme and even method. This caused some confusion and it somehow got away untested.